### DISCUSSION TITLE
Pixelate graphics with GPU plugin like in vanilla OSRS

### DISCUSSION THREAD

**Discussion by Lerroxy** [runelite/runelite#16297] [Ideas]
I really enjoy the pixelated look of OSRS without GPU or any other graphics plugin turned on, but the increased render distance of GPU plugin along with performance enhancement are too good to trade for aesthetics.

I don't really know how the plugin works, but I imagine that if not with help of GPU plugin, then it should at least be possible to do by rendering the game in a lower resolution (min window size) and then stretching it to fullscreen with "stretched mode" plugin.

Would be happy to hear at least why that wouldn't be possible or to find out that I'm in such a minority that it's not worth spending time on it xd

Thank you!

**Comment by Espionage724** [comment]
I'd be interested in this too! I forgot what the vanilla Java client looked like, but I recently re-installed it and it's definetly more blocky and jagged, but in a familiar and cool way. I could see myself using the less-sharp textures with unlocked FPS for smooth camera rotation.
